### DISCUSSION TITLE
Discord Server discussion and ideas

### DISCUSSION THREAD

**Discussion by imartinez** [imartinez/privateGPT#508] [Ideas]
Creating a Discord server would bring a more synchronous collaboration tool, allowing the community to discuss ideas, take decisions and build a stronger bond and project.

On the other hand it would move a lot of the discussions there, in a less organized way than GitHub. Creating the nth AI Discord server would also contribute to AI open community fragmentation, which is not helping the community. And lastly it would require a big amount of management, moderation, etc. to keep it useful and healthy.

I would love to get your thoughts about the possibility of creating a Discord server for privateGPT project and, in the case you think we should do it, if you'd be willing to help manage it and how, along with any best practice you'd like us to bring to it. 

Thanks in advance!

**Comment by sime2408** [comment]
beam me up Scotty 🚀 , just add link to readme
